packages/lib/server/repository/workflow.ts

```diff
@@ -139,36 +139,64 @@ export class WorkflowRepository {
 
     let verifiedEmails: string[] = [userEmail];
 
+    const secondaryEmails = await prisma.secondaryEmail.findMany({
+      where: {
+        userId,
+        emailVerified: {
+          not: null,
+        },
+      },
+    });
```

Comment:
Since secondary emails go through a verification process we should also include these in the option. Ex if someone wants to add a "support@team.com" email to the replyTo

Reply:
I also moved this to the top of the function so the user see's their email and secondary emails first.